ui/app/dashboard/directives/pnc-my-group-builds-panel/pncMyGroupBuildsPanel.js

```diff
@@ -15,63 +15,94 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-'use strict';
 
 (function () {
+  'use strict';
 
-  var module = angular.module('pnc.dashboard');
+  angular.module('pnc.dashboard').component('pncMyGroupBuildsPanel', {
```

Comment:
I think it is more reasonable to define this panel(and also the builds panel on the dashboard) as a component instead of a directive because it is a self-maintained, individual part on the page.